<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_AjoutElementPanier</class>
 <widget class="QWidget" name="D_AjoutElementPanier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>503</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>503</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ImagesIcones.qrc">
    <normaloff>:/Paiement.png</normaloff>:/Paiement.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="7">
    <widget class="QComboBox" name="CBx_ChoixPrerempli"/>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choix pré-rempli :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="Lb_Membre">
     <property name="text">
      <string>Membre :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="11">
    <widget class="QGroupBox" name="Gb_Elements">
     <property name="title">
      <string>Saisie de l'élement à ajouter au panier</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="Lb_Champs2">
        <property name="text">
         <string>Champs2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="SBx_Montant">
        <property name="maximum">
         <double>99999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Lb_Champs1">
        <property name="text">
         <string>Champs1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="SBx_Champs1">
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="SBx_Champs2">
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type de ventilation :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Montant</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="CBx_TypeVentilation"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="Le_Description"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QPushButton" name="Bt_Ajouter">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>A&amp;jouter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QPushButton" name="Bt_Annuler">
     <property name="text">
      <string>&amp;Annuler</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
